webserver/restapi.py

```
@@ -1,27 +1,150 @@
-# restblueprint.py
-from flask import Blueprint, jsonify, request
+from hmac import compare_digest
```

Ruff (0.12.2)

`hmac.compare_digest` imported but unused

Remove unused import: `hmac.compare_digest`

(F401)